lib/Stats.js

```diff
@@ -788,6 +795,25 @@ class Stats {
 						);
 					}
 				}
+				if (showChunkRootModules) {
+					const rootModules = chunkGraph.getChunkRootModules(chunk);
+					obj.rootModules = rootModules
+						.slice()
+						.sort(sortRealModules)
+						.filter(createModuleFilter("root-of-chunk"))
```

Comment:
We can improve perfarmance
```js
.slice()
.sort(sortRealModules)
.filter(createModuleFilter("root-of-chunk"))
```
=>
```js
.filter(createModuleFilter("root-of-chunk"))
.sort(sortRealModules)
```

Reply:
No because `createModuleFilter` is impure and requires the modules to be sorted